magento/pwa-studio
# [bugfix]: Adds white stroke to checkout button css in mini cart

## Description

We updated icon styles but missed the checkout button. This uses the new style, specifically `--stroke` to make the icon stroke white.

## Related Issue





Closes [PWA-506](https://jira.corp.magento.com/browse/PWA-506).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add an item to your cart.
2. Verify the mini cart checkout button shows the icon to the left as white outline on a black background.

## Screenshots / Screen Captures (if appropriate)

[![Image from Gyazo](https://i.gyazo.com/c45242aa5c629d9ab652b5f0406ad5b0.png)](https://gyazo.com/c45242aa5c629d9ab652b5f0406ad5b0)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
